### PR TITLE
Adding whitespace to each file to make the code review simpler

### PR DESCRIPTION
**Pull request by mpretty-cyro**

### CODE REVIEW

File: yourflix/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="12121" systemVersion="16G29" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="49e-Tb-3d3">
-    <device id="retina4_7" orientation="portrait">
-        <adaptation id="fullscreen"/>
-    </device>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="19529" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="49e-Tb-3d3">
+    <device id="retina4_7" orientation="portrait" appearance="light"/>
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="12089"/>
+        <deployment identifier="iOS"/>
```
Comment (mpretty-cyro) [inline yourflix/Base.lproj/Main.storyboard:5]: The collection views don't current have any padding from the left/right edges of the screen - please add some in (eg. 10pt)

File: yourflix/MoviesController.swift [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
+    let dataLoader = MoviesDataLoader(resource: "movies")
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        guard let movies =  dataLoader.load() else {
```
Comment (mpretty-cyro) [inline yourflix/MoviesController.swift:14]: Since the data is currently hard-coded this loading of data can probably be shifted to `viewDidLoad` (which will occur once the first time the `view` property is accessed), putting this `load` method in `viewWillAppear` will result in the data being re-loaded in every time the user swaps to this screen

File: yourflix/TVViewController.swift [hunk 3]
```diff
@@ -1,59 +1,59 @@
...
+    let dataLoader = TVDataLoader(resource: "tv")
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        guard  let shows = dataLoader.load() else {
```
Comment (mpretty-cyro) [inline yourflix/TVViewController.swift:14]: Same `viewDidLoad` comment as the `MoviesViewController`

**Review by mpretty-cyro — COMMENTED** [review]
I'd generally plug in swiftlint to have warnings appear in XCode for simple formatting discrepancies.
